crates/bevy_pbr/src/light.rs

```diff
@@ -721,9 +721,7 @@ fn compute_aabb_for_cluster(
     let p_min = ijk.xy() * tile_size;
     let p_max = p_min + tile_size;
 
-    let cluster_min;
-    let cluster_max;
-    if is_orthographic {
+    let (cluster_min, cluster_max) = if is_orthographic {
```

Comment:
Disagree on this change; this is significantly less clear to me.

Reply:
This diff in context may change your mind about clarity.